invenio_search_ui/static/templates/invenio_search_ui/range.html

```diff
@@ -0,0 +1,11 @@
+<div class="cds-search-facets bg-w pt-30" ng-show="vm.invenioSearchResults.aggregations.years.buckets.length > 0">
+    <div class="cds-border ba pa-10 b-r-3">
+        <h2 class="text-capitalize f8 mt-5">
```

Comment:
we can remove this template :P we are using the on from ``invenio-search-js`` :)
```
SEARCH_UI_JSTEMPLATE_RANGE = 'node_modules/invenio-search-js/dist/templates/range.html'
```